tests/NLog.UnitTests/LayoutRenderers/MessageTests.cs

```diff
@@ -219,5 +219,34 @@ public void MessageWithExceptionAndCustomSeparatorTest()
             logger.Debug(ex);
             AssertDebugLastMessage("debug", ex.ToString());
         }
+
+        [Fact]
+        public void SingleParameterException_OutputsSingleStackTrace()
+        {
+            // Stage
+            var logFactory = new LogFactory();
+            var logConfig = new LoggingConfiguration(logFactory);
+            var logTarget = new NLog.Targets.DebugTarget("debug") { Layout = "${message}|${exception}" };
+            logConfig.AddRuleForAllLevels(logTarget);
+            logFactory.Configuration = logConfig;
+            var logger = logFactory.GetLogger("SingleParameterException");
+
+            // Act
+            Exception argException = null;
+            try
+            {
+                logger.Info("Hello");
+                throw new ArgumentException("Holy Moly");
+            }
+            catch (Exception ex)
+            {
+                logger.Fatal(ex);
+                argException = ex;
+            }
+
+            // Assert
+            Assert.NotNull(argException);
+            Assert.StartsWith($"{argException.GetType()}: {argException.Message}|{argException.GetType()}", logTarget.LastMessage);
```

Comment:
Maybe just write out the values? Those are fixed.

So

```
 Assert.StartsWith("System.ArgumentException: Holy Moly |System.ArgumentException", logTarget.LastMessage);
```

Reply:
Fixed